Quick-Start [DA]/MVP/MVP+Makefile.md

```diff
@@ -1 +1,274 @@
+# Minimum Viable Product:
+A basic tutorial on creating a PostgreSQL extension using the bare minimum files and a Makefile:
+
+- A **control file**, which tells PostgreSQL some basic information about the extension, such as its name, version, and schema.
+- A **SQL script file**, which contains the SQL commands to create the extension's objects.
+- A **Makefile** is an optional but highly recommended file that helps automate the build process of the extension. It defines the compilation and installation steps, making it easier to build and install the extension on different systems.
+
+# [CODE](https://github.com/IshaanAdarsh/Postgres-extension-tutorial/tree/main/Code/my_extension)
+
+## A Postgres extension using only a control file, an SQL file and a Makefile:
+
+### Step 1: Create the Extension Directory
+- Create a directory for your extension. Let's call it "my_extension".
+
+```bash
+$ mkdir my_extension
+$ cd my_extension
+```
+
+### Step 2: Create the Extension Control File
+- Inside the "my_extension" directory, create a file named "my_extension.control". 
+  - This file defines the metadata and dependencies of your extension.
+
+```bash
+$ touch my_extension.control
+```
+
+- Open the "my_extension.control" file and add the following content:
+
+```control
```

Comment:
I wonder if `ini` syntax would work well here.

Reply:
ERROR: does not adhere to the INI syntax
I tried using `ini` syntax in control file, got this error in my iterm shell